cmk/base/plugins/agent_based/prism_vm.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- encoding: utf-8; py-indent-offset: 4 -*-
+# (c) Andreas Doehler <andreas.doehler@bechtle.com/andreas.doehler@gmail.com>
+# This is free software;  you can redistribute it and/or modify it
+# under the  terms of the  GNU General Public License  as published by
+# the Free Software Foundation in version 2.  check_mk is  distributed
+# in the hope that it will be useful, but WITHOUT ANY WARRANTY;  with-
+# out even the implied warranty of  MERCHANTABILITY  or  FITNESS FOR A
+# PARTICULAR PURPOSE. See the  GNU General Public License for more de-
+# ails.  You should have  received  a copy of the  GNU  General Public
+# License along with GNU Make; see the file  COPYING.  If  not,  write
+# to the Free Software Foundation, Inc., 51 Franklin St,  Fifth Floor,
+# Boston, MA 02110-1301 USA.
+import ast
+from typing import Any, Dict
+
+from .agent_based_api.v1 import register
+from .agent_based_api.v1.type_defs import StringTable
+
+Section = Dict[str, Any]
+
+
+def parse_prism_vm(string_table: StringTable) -> Section:
+    parsed = {}
+    parsed = ast.literal_eval(string_table[0][0])
```

Comment:
since those are only dictionaries I prefer to use json. This follows the pattern of other newer special agents.